src/routing/lazy_messaging.rs

```diff
@@ -97,9 +103,9 @@ fn create_other_section_message(
     node: &Node,
     section: &Section,
     network: &Network,
+    variant: Variant,
```

Comment:
as mentioned above, I think we can retain the OtherSection, without introducing those new variants.